src/main/java/com/thalesgroup/hudson/plugins/cccc/CcccPublisher.java

```diff
@@ -112,4 +118,40 @@ public String getMetricFilePath() {
         return metricFilePath;
     }
 
+    @Override
+    public void perform(@Nonnull Run<?, ?> run, @Nonnull FilePath filePath, @Nonnull Launcher launcher, @Nonnull TaskListener taskListener) {
```

Comment:
This should replace the old perform method iirc, not added as an extra.

Reply:
Thanks, I'll change that!